RestApi/Sepes.RestApi/Services/PodService.cs

```diff
@@ -112,9 +128,27 @@ private async Task RemoveNetworkSecurityGroup(Pod newPod)
 
         private Dictionary<ushort, string[]> GenerateRuleDictionary(IEnumerable<Rule> array)
         {
-            var g = array.GroupBy(r => r.port, r => r.ip, (port, ips) => new { Key = port, Value = ips});
+            var g = array.GroupBy(r => r.port, r => r.ip, (port, ips) => new { Key = port, Value = ips });
             var ruleDict = g.ToDictionary(r => r.Key, r => r.Value.ToArray());
             return ruleDict;
         }
+
+        private async Task<(string oldName, string newName)> GetNSGName(string networkSecurityGroupName)
```

Comment:
I don't like how this method is being used 2 different places in 2 different ways. It can be a bit confusing.